proto/lighthouse-result.proto

```diff
@@ -42,6 +42,10 @@ enum LighthouseError {
   PARSING_PROBLEM = 15;
   // The trace data failed to stream over the protocol.
   READ_FAILED = 16;
+  // Used when security error prevents page load.
+  INSECURE_DOCUMENT_REQUEST = 17;
+  // Used when security error prevents page load.
+  PROTOCOL_TIMEOUT = 18;
```

Comment:
I removed REQUEST_CONTENT_TIMEOUT ... instead of making a new enum value for `PROTOCOL_TIMEOUT`, could we just reuse `REQUEST_CONTENT_TIMEOUT`'s enum (rename it)? Would that be OK for proto compat?

Reply:
nope, can't rename. 

But actually looks like we never exposed REQUEST_CONTENT_TIMEOUT in the proto. So I think we're OK. :)